content/en/helm/_index.md

```diff
@@ -0,0 +1,143 @@
+---
+title: 'Helm Chart Deployment'
+date: 2023-01-26T13:00:00+07:00
+description: 'Use Helm Charts to deploy Tetrate Istio Distro'
+# type dont remove or customize
+type: 'docs'
+weight: 35
+---
+
+### Helm Charts
+
+While there are multiple benefits of using `getmesh` CLI tool, there are use-cases that require a complete automation and repeatable approach. To address this need, Tetrate offers a de-facto standard approach with Helm charts. This page illustrates typical steps to deploy `Tetrate Istio Distro` via Helm.
+
+#### Add Tetrate Helm Repo
+
+Tetrate hosts publicly available Helm Repo that can be accesses via standard Helm commands:
+
+```bash
+helm repo add tid https://tetratelabs.github.io/istio-helm/
+helm repo update tid
+```
+
+  Output:
+  
+  ```pre
+   $ helm repo add tid https://tetratelabs.github.io/istio-helm/
```

Comment:
I see now the chart repo is different, compared to [this one](https://github.com/tetratelabs/helm-charts/blob/main/README.md#setup-helm-repo), which one is the official we need to provide to our TID customers?

Reply:
thanks @nauticalmike 
the repo which is mentioned in this PR can be updated automatically while the one you're linking to is semi-manual work.
as for `official` - if you know who decides - we can ask that person :-)